test/e2e/kind.sh

```diff
@@ -23,7 +23,7 @@ kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
 nodes:
   - role: control-plane
-    image: kindest/node:v1.25.3@sha256:f52781bc0d7a19fb6c405c2af83abfeb311f130707a0e219175677e366cc45d1
+    image: kindest/node:v1.30@sha256:f52781bc0d7a19fb6c405c2af83abfeb311f130707a0e219175677e366cc45d1
```

Comment:
We should also update the `sha256` reference here. Maybe we could use the latest `v1.30.10`:
https://hub.docker.com/layers/kindest/node/v1.30.10/images/sha256-e382f9b891474f1c4b0b5cfcf27f8e471f1bdc1f285afe38adeec1bd5b856cfe